.github/workflows/sycl-linux-precommit.yml

```diff
@@ -110,6 +110,8 @@ jobs:
             use_dev_igc: ${{ contains(needs.detect_changes.outputs.filters, 'devigccfg') }}
             extra_lit_opts: --param matrix-xmx8=True --param gpu-intel-dg2=True
             env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
+            # Run only if the PR does not have the 'ci-no-devigc' label.
+            skip_run: ${{ contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') && 'true' || 'false' }}
```

Comment:
Do we still need the ( && 'true' || 'false')?

Reply:
Yes, because `skip_run` is of type String

Reply:
Actually, you are right. Now that I refer to GHA docs (https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains), `contains()` return `true`/`false` strings. Previously, I though `contains()` returns boolean values.